### DISCUSSION TITLE
Is there a way to run "panda codegen" in Node.js 14 ?

### DISCUSSION THREAD

**Discussion by ryanpark** [chakra-ui/panda#1515] [General]
Hi Panda 🐼

Unfortunately, my app is running on Node.js 14, and I'm unable to upgrade to a higher version. 
Is there a way to run `panda codegen` in Node.js 14? I'm currently facing the following error:

```   
config.cwd ||= runtime.cwd();
SyntaxError: Unexpected token '||='
```
which is the logical OR assignment operator does not support in node 14.

I'm also exploring the option of using `@babel/plugin-transform-logical-assignment-operators,`
but I'm not entirely sure how to implement it.

**Comment by ryanpark** [comment]
fixed

**Reply by anubra266** [reply]
@ryanpark Could you share your solution, for anyone who gets to this in the future.
